### PR TITLE
Nicolas - SF Automation - Master - LPD-36648 

### PR DESCRIPTION
**Pull request by MicaelleSilvaa**
Ticket: https://liferay.atlassian.net/browse/LPD-36648

### CODE REVIEW

**Comment by MicaelleSilvaa** [comment]
@nicolas-sss

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Review by nicolas-sss — APPROVED** [review]

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 1]
```diff
@@ -711,6 +711,13 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		contact.setSuffixListTypeId();
 	}
 
+	public void method(
+		DDMFormViewFormInstanceRecordsDisplayContext
+			ddmFormViewFormInstanceRecordsDisplayContext){
+		
+		ddmFormViewFormInstanceRecordsDisplayContext.getSearchContainer();
+	}
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:719]: @MicaelleSilvaa this breakline was the SF or you?

Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:719]: This breakline was the SF.

Comment (nicolas-sss) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:719]: @MicaelleSilvaa okay then, Do a rebase, fix the conflicts locally, and then resend the changes. Once the changes pass the tests again the PR is approved and you will be free to proceed with forwarding.

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by MicaelleSilvaa** [comment]
forward to Brian: https://github.com/brianchandotcom/liferay-portal/pull/154548

**Pull request closed without merging**